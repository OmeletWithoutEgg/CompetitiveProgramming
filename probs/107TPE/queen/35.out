-151983.00 -26406840.00
