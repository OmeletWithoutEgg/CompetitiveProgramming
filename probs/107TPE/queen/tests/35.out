16778378.29 37592012.18
